### DISCUSSION TITLE
Cypress  "npm i -D cypress-soft-assertion" support is not available in Cypress 13 version

### DISCUSSION THREAD

**Discussion by Aditya2680** [cypress-io/cypress#28967] [Questions and Help]
Cypress Soft assert plugin "npm i -D cypress-soft-assertion"  was working till cypress 11. How we can achieve the soft assertion in Cypress 13.

**Comment by portyu9** [comment]
Hi @Aditya2680 

This is not a Cypress 13 regression in a Cypress-owned API.

`cypress-soft-assertions` is a **third-party plugin**, and its own documentation explicitly says that it works with Cypress v11 and does not support Cypress retry behavior.

So installing:

```bash
npm i -D cypress-soft-assertions
```

does not provide a supported soft-assertion API for Cypress 13.

There is no Cypress configuration flag that can restore compatibility with that package. You would need to use a soft-assertion implementation that explicitly supports your Cypress version, update/fork the old plugin, or implement assertion aggregation yourself.

I would not recommend downgrading Cypress solely to keep this old plugin unless you are temporarily maintaining a legacy test suite.

Please mark this answer as accepted if it helped, thank you.
